assets/js/stripe.js

```diff
@@ -879,43 +807,50 @@ jQuery( function( $ ) {
 		},
 
 		/**
-		 * Queries the server for a PaymentIntent.
-		 *
-		 * @param {string} savedCard The ID of the saved card. (Optional)
-		 * @return {Promise} A promise that resolves to a payment intent.
+		 * Handles changes in the hash in order to show a modal for PaymentIntent confirmations.
 		 */
-		getIntent: function( savedCard ) {
-			savedCard = savedCard || 'new';
+		onHashChange: function() {
```

Comment:
The hash approach is cool but might be difficult to immediately understand for the next maintainer. Please document how it works either in a few comments here or in a README.md in this repo or something.